plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -202,14 +202,10 @@ def _process_searchspace_config(self) -> None:
                         """
                     ),
                 )
-                self.params = OmegaConf.create(
```

Comment:
This had to change back to a dict type since creating OmegaConf configuration with real objects need the 'allow_objects' flag. This flag is part of the internal API as per [this code](https://github.com/omry/omegaconf/blob/6a4fd518d53c7b9996b720458352b1e6a643e769/omegaconf/nodes.py#L146).